chroma-common/LICENSE.txt

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2016 Intel Corproration. All Rights Reserved.
```

Comment:
License needs to be fixed.

We should remove this file until it lives in it's own package. At which point it would make sense to add the MIT license.